Doc/library/tkinter.rst

```diff
@@ -11,6 +11,11 @@ the Tk GUI toolkit.  Both Tk and :mod:`Tkinter` are available on most Unix
 platforms, as well as on Windows systems.  (Tk itself is not part of Python; it
 is maintained at ActiveState.)
 
+Running ``python -m Tkinter`` from the command line should open a window
+demonstrating a simple Tk interface, letting you know that :mod:`tkinter` is
```

Comment:
The only change needed before merging this is `Tkinter` instead of `tkinter` here